src/lib/vrf_lib/standalone.ml

```diff
@@ -77,15 +77,15 @@ module Make
     end) (Message : sig
       open Impl
 
-      type t [@@deriving sexp]
```

Comment:
Why this change out of curiosity?

Reply:
This was to make it match the types that we pass to the `integrated` version, so that I didn't have to recreate the module arguments with slightly different layouts for each.